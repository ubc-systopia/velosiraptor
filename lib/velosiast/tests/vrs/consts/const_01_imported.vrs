import const_00_defs;

const BAR1: int = 1234;
const BAR2: int = 0xabcd;
const BAR3: int = FOO1 + FOO2;