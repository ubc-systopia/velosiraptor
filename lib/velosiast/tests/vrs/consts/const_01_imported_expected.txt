VelosiAst(tests/vrs/consts/const_01_imported.vrs)
--------------------------------------------------------

const FOO1 : int = 0x7b;

const FOO2 : int = 0x1234;

const FOO3 : size = 0x12af;

const BAR1 : int = 0x4d2;

const BAR2 : int = 0xabcd;

const BAR3 : int = 0x12af;
--------------------------------------------------------
